vms/proposervm/post_fork_option_test.go

```diff
@@ -154,6 +157,9 @@ func TestBlockVerify_PostForkOption_CoreBlockVerifyIsCalledOnce(t *testing.T) {
 	// Verify an option once; then show that another verify call would not call coreBlk.Verify()
 	coreVM, _, proVM, coreGenBlk, _ := initTestProposerVM(t, time.Time{}, 0)
 	proVM.Set(coreGenBlk.Timestamp())
+	defer func() {
```

Comment:
do we need this to be wrapped in another func? i.e can't we just use `defer require.NoError(proVM.Shutdown(context.Background()))`

Reply:
We don't need the wrap. Cleaned it up. Thanks

Reply:
Actually we do need the wrap to avoid immediate argument evaluation. Reverted